pkgs/by-name/xf/xfs_undelete/package.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, makeWrapper
+, coreutils
+, tcl-8_6
+, tcllib
+}:
+
+stdenv.mkDerivation {
+  pname = "xfs_undelete";
```

Comment:
While there's nothing in the way of having derivations named with underscore, the package attribute name unfortunately has some conventions. I think you'll have to rename the directory to `xfs-undelete`, and redo the commit message as well. This `pname` should stay the same AFAIK.

Reply:
> Consider moving the package to `pkgs/by-name/xf/xfs_undelete/package.nix`.

Note, that I have followed your exact suggestion : -)

Reply:
Yeah I know, sorry about that. I was a bit quick and overlooked it.

The files and directories with dashes thing is specified in CONTRIBUTING.md, so I think it's sort of a hard requirement: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#file-naming-and-organisation